### PR TITLE
Skrell humanitarian ship

### PR DESCRIPTION
**Pull request by meep109**
augh

### CODE REVIEW

**Comment by meep109** [comment]
!review

**Comment by FluffyGhoster** [comment]
`Duplicated APCs in area: Nralakk Humanitarian Ship - Fore Hallway. #1: The Nralakk Humanitarian Ship - Fore Hallway APC (/obj/machinery/power/apc/north) ([0x201f71b]) (the shuttle floor) (123,124,18) (/turf/simulated/floor/shuttle/skrell)  #2: The Nralakk Humanitarian Ship - Fore Hallway APC (/obj/machinery/power/apc/east) ([0x201f870]) (the shuttle floor) (127,154,18) (/turf/simulated/floor/shuttle/skrell) → @@@code/unit_tests/power_tests.dm:65`

**Comment by DreamySkrell** [comment]
image
shuttle cannot fit in its transit area

image
should be like this (the shuttle here removed of course)

also transit area should use `/turf/space/transit` turfs
here specifically `/turf/space/transit/north`

**Comment by DreamySkrell** [comment]
image
docking and shuttle airlock markers should be defined in code now, not in the map
see sol yacht or scc scout for a example

**Comment by DreamySkrell** [comment]
image
missing vent/scrubber in this docking arm
also the canister has no port under it

**Comment by DreamySkrell** [comment]
image
wrong orientation for these firelocks, they are north-south instead of east-west

**Comment by DreamySkrell** [comment]
image
misplaced pipe

**Comment by DreamySkrell** [comment]
image
image
image
wires

**Comment by DreamySkrell** [comment]
image
shuttle has `/obj/machinery/power/smes/buildable/horizon_shuttle` when it should be `/obj/machinery/power/smes/buildable/third_party_shuttle`

**Comment by DreamySkrell** [comment]
image
do not put vents/scrubbers on top of doors or firedoors
they're not going to work when the firedoor closes

**Comment by DreamySkrell** [comment]
image
this large multi-tile tank needs a console to operate it
use large tank markers, see example in sol yacht or scc scout

**Comment by DreamySkrell** [comment]
image
all away site/ship maps should be 255x255

**Comment by DreamySkrell** [comment]
image

use `/obj/effect/map_effect/window_spawner/full/shuttle` for window spawners
see sol yacht or other newer maps for an example

**Comment by DreamySkrell** [comment]
image
image
pixel-shift things on tables so they visually look like they're actually on tables, and not half floating in the air

image
for example like here

image
do not put multiple things on top of each other like this, especially machinery, where you can't see the thing below

File: maps/away/ships/fed_humanitarian/fed_humanitarian_ship_ghostroles.dm [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/datum/ghostspawner/human/fed_humanitarian
+	short_name = "fed_humanitarian"
+	name = "Nralakk Humanitarian Worker"
+	desc = "You are a worker aboard a Nralakk Federation humanitarian vessel, sent to aid the Izweski Hegemony."
```
Comment (DreamySkrell) [inline maps/away/ships/fed_humanitarian/fed_humanitarian_ship_ghostroles.dm:4]: every ghostspawner needs `welcome_text` with more information about what you are supposed to do after spawning there, any other non-obvious stuff about the ship or the role, or anything else
see scc scout or sol yacht for an example

File: maps/away/ships/fed_humanitarian/fed_humanitarian_ship.dm [hunk 2]
```diff
@@ -0,0 +1,138 @@
+/datum/map_template/ruin/away_site/fed_humanitarian_ship
+	name = "Nralakk Humanitarian Ship"
+	description = "Luxupi-class freighters are one of the most common civilian vessels of the Nralakk Federation. Usually outfitted with midsized warp drives, they transport all manner of goods and cargo across the Nralakk Federation - sometimes even to other nations within the Orion Spur. These ships are capable of atmospheric operations depending on size, with the smaller vessels being used as intermediaries between larger freighters and planets. Freighters that do not travel outside of the inner systems do not normally have weapons, but will be outfitted with basic weaponry if travelling into known Marauder hotspots within the Traverse."
+	prefix = "ships/fed_humanitarian/"
+	suffixes = list("fed_humanitarian_ship.dmm")
+	sectors = list(SECTOR_UUEOAESA)
+	spawn_weight = 10
+	ship_cost = 1
+	id = "skrell_humanitarian"
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
```
Comment (DreamySkrell) [inline maps/away/ships/fed_humanitarian/fed_humanitarian_ship.dm:10]: .

File: maps/away/ships/fed_humanitarian/fed_humanitarian_ship.dm [hunk 3]
```diff
@@ -0,0 +1,138 @@
+/datum/map_template/ruin/away_site/fed_humanitarian_ship
+	name = "Nralakk Humanitarian Ship"
+	description = "Luxupi-class freighters are one of the most common civilian vessels of the Nralakk Federation. Usually outfitted with midsized warp drives, they transport all manner of goods and cargo across the Nralakk Federation - sometimes even to other nations within the Orion Spur. These ships are capable of atmospheric operations depending on size, with the smaller vessels being used as intermediaries between larger freighters and planets. Freighters that do not travel outside of the inner systems do not normally have weapons, but will be outfitted with basic weaponry if travelling into known Marauder hotspots within the Traverse."
+	prefix = "ships/fed_humanitarian/"
+	suffixes = list("fed_humanitarian_ship.dmm")
+	sectors = list(SECTOR_UUEOAESA)
+	spawn_weight = 10
```
Comment (DreamySkrell) [inline maps/away/ships/fed_humanitarian/fed_humanitarian_ship.dm:7]: why?

**Comment by DreamySkrell** [comment]
image
I am not going to request changes to make this map more pretty
but it is kinda bland and very much lacks decoration
see the tiqraqki ship for inspiration

**Comment by DreamySkrell** [comment]
image
crew quarters should be nicer
like this is a nralakk fed vessel, from one of the richest factions in the spur, they should afford to have nice crew quarters
does not have to be a big room for every one crew member, but like two rooms for 3 crew each would be fine I think

also where would they house the people they give humanitarian aid, if they can't leave them on mohes or wherever?
should there not be some like, quarters or something for any visitors? a communal room or something?

**Comment by DreamySkrell** [comment]
image
this is kinda tight and should have another entrance to the south

**Comment by DreamySkrell** [comment]
image
do not put machines on top of walls
(do not put it on a floor and pixel-shift into a wall, either)

**Comment by DreamySkrell** [comment]
image
(this applies to engineering but to other areas as well)
this engineering section is kinda super clean and does not really look like a place that people live in and use everyday

there should be a bit of dirt on the floor, tables with random tools on them, crates, other random gear
I hate to be like "look at my maps for inspiration" cause it feels very, idk, vain I guess

image
but for example look at Tret that I added recently, I do think it feels very lived in like if it was used by someone in their day-to-day work and life

also there should be a wall between the thruster and the interior of the ship, see other new maps added recently

**Comment by DreamySkrell** [comment]
image
needs east and west docks

**Comment by DreamySkrell** [comment]
image
as this is a humanitarian ship, the shuttle should have designated cargo space, and bigger passenger capacity with more seats

**Comment by DreamySkrell** [comment]
image
remove the CIC, this is not a combat ship

**Pull request closed without merging**